hazelcast/src/test/java/com/hazelcast/jet/core/JetTestSupport.java

```diff
@@ -404,15 +404,21 @@ public void cleanUpCluster(HazelcastInstance... instances) {
      * retry.
      */
     public static void ditchJob(@Nonnull Job job, @Nonnull HazelcastInstance... instances) {
+        //Cancel the job on cluster members
         ditchJob0(job, instances);
+
         // Let's wait for the job to be not RUNNING on all the members.
         assertTrueEventually(() -> {
-            assertNotEquals(RUNNING, job.getStatus());
-            for (HazelcastInstance instance : instances) {
-                Job instanceJob = instance.getJet().getJob(job.getId());
-                if (instanceJob != null) {
-                    assertNotEquals(RUNNING, instanceJob.getStatus());
+            //If a member throws exception ignore it
+            try {
+                assertNotEquals(RUNNING, job.getStatus());
+                for (HazelcastInstance instance : instances) {
+                    Job instanceJob = instance.getJet().getJob(job.getId());
+                    if (instanceJob != null) {
+                        assertNotEquals(RUNNING, instanceJob.getStatus());
+                    }
                 }
+            } catch (Exception ignored) {
```

Comment:
Maybe we should log the exception here? I would also prefer to try/catch in the ```for``` loop. Right now if one member throws we do wait for stopping the job on the other member.

Reply:
I have changed the code to catch exception for each member check